### ISSUE TITLE
Namedtuple with attribute with trailing underscore

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1462]
Hi,

The following piece of code:

```
#!python

X = collections.namedtuple(
    'X',
    ('subject', 'from_', 'to', 'body',)
)

```

gives me

```
Warning, treated as error:
/xxxx.py:docstring of xxx.X:1: ERROR: Unknown target name: "from".
```

when processing it with Sphinx-

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1462
- Originally reported by: Anonymous
- Originally created at: 2014-05-08T17:24:37.516

**Comment by JelteF** [comment]
I have this same problem. For me it only occurs when autosummary and numpydoc are enabled. In that case it happens to every argument with an underscore at the end.

**Comment by matthew-brett** [comment]
I'm interested in this issue - I think I have it too.  Would one of you mind generating a minimal reproducible example, maybe in the form of a little git repo for which `sphinx-build` generates this warning?

**Comment by matthew-brett** [comment]
I put up a template repo for testing here : https://github.com/matthew-brett/smallpkg

You'll see there my attempts to make this warning happen, but no success so far.

**Comment by tk0miya** [comment]
Fixed at 8d96c90.
Thank you for reporting
